src/utils/client.js

```diff
@@ -0,0 +1,123 @@
+const jose = require('node-jose');
+
+const axios = require('axios');
+const { v4 } = require('uuid');
+const servers = require('../utils/serverConfig.json');
+const clientId = 'medmorph_backend';
+const keys = require('./privateKey.json');
+const queryString = require('query-string');
+
+class Client {
+  /**
+   *
+   * @param {*} jwks This clients private key set used for signing requests
+   */
+  constructor(jwks) {
+    this.jwks = jwks;
+  }
+
+  /**
+   * Generate and return access token for the specified server
+   * @param {*} server  the server to get an access token for
+   */
+  async connectToServer(server) {
+    if (server === 'EHR') {
```

Comment:
Let's make this function take in a URL directly so that we don't have to maintain a separate list to look items up in

Reply:
This lets people put in whatever they want as opposed to a limited set that we known will work, is this an issue?

Reply:
That's fine. In practice the connections are going to be made only to selected "known good" servers anyway, it's not like we're going to have a UI that users type in free text into